src/GitLabApiClient/Models/MergeRequests/Responses/MergeRequestDiffVersion.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using GitLabApiClient.Models.Users.Responses;
+using Newtonsoft.Json;
+
+namespace GitLabApiClient.Models.MergeRequests.Responses;
+
+public class MergeRequestDiffVersion
+{
+    [JsonProperty("id")]
+    public int Id { get; set; }
+
+    [JsonProperty("head_commit_sha")]
+    public string HeadCommitSha { get; set; }
+
+    [JsonProperty("base_commit_sha")]
+    public string BaseCommitSha { get; set; }
+
+    [JsonProperty("start_commit_sha")]
+    public string StartCommitSha { get; set; }
+
+    [JsonProperty("created_at")]
+    public DateTime CreatedAt { get; set; }
```

Comment:
making sure it can't be null?

Reply:
It shouldn't, not according to their docs.